0009-enpoint-events-ipc/README.md

```diff
@@ -0,0 +1,126 @@
+---
+name: Endpoint events IPC
+shortname: 9/ENDPOINT_EVENTS
```

Comment:
No idea :)

`EPE`, `EE`, `EPEI`, `EEI`, `EEIPC`, `EPEIPC`. Pick whatever you like? (But I would live the question for @hunch7.)